src/include/migraphx/op/common.hpp

```diff
@@ -15,6 +17,19 @@ enum padding_mode_t
     valid
 };
 
+// The pooling modes must correspond 1-1 to the operators defined for struct parse_pooling.
+enum class pooling_mode
+{
+    avg,
```

Comment:
I think it would be better to name this `average` so it matches the printed output.

Reply:
I'll pass on making that change--I already changed it once.

Reply:
Whats the issue with making the change?